politeiad/cmd/legacypoliteia/proposal.go

```diff
@@ -28,9 +28,8 @@ type proposal struct {
 	// usermd plugin
 	StatusChanges []usermd.StatusChangeMetadata
 
-	// ticketvote plugin. These fields may be nil depending on the proposal,
-	// i.e. abandoned proposals will not have ticketvote data.
-	VoteMetadata *ticketvote.VoteMetadata
+	// ticketvote plugin
+	VoteMetadata ticketvote.VoteMetadata
```

Comment:
Don't change this.  A VoteMetadata is not going to exist for every proposal.

Reply:
oh my bad, rebase issues